packages/react-ui-core/src/Listing/ListingComponents.js

```diff
@@ -66,9 +66,13 @@ Ratings.contextTypes = {
   rating: PropTypes.object,
 }
 
-const Photos = (props, { photos }) => (
-  photos ? <PhotoCarousel {...props} items={photos} /> : null
-)
+const Photos = (props, { photos }) => {
```

Comment:
This looks to do exactly the same thing. Honestly just curious why this is preferable to the ternary